framework/simd.conf

```diff
@@ -168,12 +182,15 @@ arch=WLC	SNC	shstk,movdiri,movdir64b,ibt,keylocker		# avx512vp2intersect
 arch=GLC	WLC	avx512bf16,avxvnni,cldemote,waitpkg,serialize,uintr	# pks,tsxldtrk
 arch=RPC	GLC
 arch=RWC	RPC	avx10,prefetchiti
+arch=LNC	RWC	cmpccxadd,lam,avxifma,avxneconvert,avxvnniint8,sha512,sm3,sm4
```

Comment:
Move the crypto instructions to a comment: they may be optional in some SKUs (see pclmul, aesni and shani above). The script currently has no way to declare optional content.

Repeats below.